tests/python/lib/testcase.py

```diff
@@ -107,6 +107,20 @@ def custom_setup(cls):
     def custom_teardown(cls):
         pass
 
+    def setup_method(self, method):
+        self.custom_setup_method(method)
+
+    def teardown_method(self, method):
+        self.custom_teardown_method(method)
+
+    @classmethod
+    def custom_setup_method(cls, method):
+        pass
+
+    @classmethod
+    def custom_teardown_method(cls, method):
+        pass
```

Comment:
These methods shouldn't be static
Similar as basic `setup_method` and `teardwon_method`

Reply:
fix